### PR TITLE
Fix for CoinCollect adapter

### PR DESCRIPTION
**Pull request by r0b0roach**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
CoinCollect

##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/coincollect exports TVL: 
        
 
 
    polygon-staking           66.43 k
    polygon                   29.28 k
    staking                   66.43 k
    
    total                    29.28 k

**Comment by r0b0roach** [comment]
> The adapter at projects/coincollect exports TVL:
> 
> ```
> polygon-staking           66.43 k
> polygon                   29.28 k
> staking                   66.43 k
> 
> total                    29.28 k 
> ```

Correct

File: projects/coincollect/index.js [hunk 1]
```diff
@@ -10,8 +10,7 @@ module.exports = {
     incentivized: true,
     polygon: {
         tvl: getUniTVL({ factory: '0x2bc17223A99B6e2857796a5F64A1ED91067b5657', useDefaultCoreAssets: true, fetchBalances: true }),
-        staking: sumTokensExport({owner: MasterChefContract, tokens: [COLLECT], lps: ['0x0cCc84b6506003487AEC687085e82C2f912E607B'], useDefaultCoreAssets: true,  }),
+        staking: stakingUnknownPricedLP(MasterChefContract, COLLECT, "polygon", "0x0cCc84b6506003487AEC687085e82C2f912E607B"),
```
Comment (g1nt0ki) [inline projects/coincollect/index.js:13]: `stakingUnknownPricedLP`  is a legacy method and doesnt take into account amount of liquidity,  for example, the pool can have only 1$ worth of tokens and can still be used to as price oracle, thats why we dont use it for new adapters.

sumTokensExport method is bit more careful, think the it allows token value not to exceed 5x (or 10x?) than the liquidity in LPs provided.

Comment (g1nt0ki) [reply projects/coincollect/index.js:13]: Solution here might be getting the token listed on coingecko, we pull token prices from there.

**Pull request closed without merging**